### PR TITLE
ci: add verify-package-version

### PR DESCRIPTION
**Pull request by xrkffgg**
### 背景

https://github.com/ant-design/ant-design/pull/29198#issuecomment-772212716

### 说明
```yml
# 新增 PR title 只有包含 changelog 才进行触发，避免所有 PR 都触发
if: contains(github.event.pull_request.title, 'changelog')
```

```yml
# title 包含校验。此处为 docs
title-include-content: 'docs'
# title 包含版本校验，不过默认就是 true。这里会根据 PR 里的 package.json 中 version 和 title 对比。支持 Fork 的 PR
title-include-version: true
# 是否进行评论。Fork 的 PR 无权限，不会进行这个操作，只会纯校验。在 CI 显示结果。
open-comment: true
```

### 图示

image

image

image

image

image

### CODE REVIEW

**Comment by afc163** [comment]
版本号只能远端取么，本地有代码库应该容易拿吧，走请求容易失败

https://github.com/actions-cool/verify-package-version/blob/ec01ff9f4f8c0c0363981e16ae672e28e5ebf0d0/src/main.js#L36

**Comment by xrkffgg** [comment]
我试试

**Comment by xrkffgg** [comment]
@afc163 O了

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "antd",
-  "version": "4.12.1",
+  "version": "4.12.2",
```
Comment (xrkffgg) [inline package.json:3]:
```suggestion
  "version": "4.12.1",
```

**Pull request merged**